mlrun/api/api/endpoints/feature_store.py

```diff
@@ -397,6 +397,46 @@ def ingest_feature_set(
     )
 
 
+@router.post(
+    "/projects/{project}/feature-sets/{name}/references/{reference}/publish",
+    response_model=schemas.FeatureSetPublishOutput,
+)
+def publish_feature_set(
+    project: str,
+    name: str,
+    reference: str,
+    publish_tag: str = Query(None, alias="publish_tag"),
```

Comment:
change to - (publish-tag)
```suggestion
    publish_tag: str = Query(None, alias="publish-tag"),
```